### DISCUSSION TITLE
How to perform stress testing

### DISCUSSION THREAD

**Discussion by huangtao730** [open-telemetry/opentelemetry-java-instrumentation#12913] [Q&A]
How to perform stress testing opentelemetry-java-instrumentation？？Is there any specific method?

**Comment by steverao** [comment]
> How to perform stress testing opentelemetry-java-instrumentation?

I don't understand your question fully. Can you introduce more about it?
If it's about measuring overhead of OTel Java Agent. You can compare the overheads of your testing application with 
 and without using an Agent. We also provided some general [explanations](https://opentelemetry.io/docs/zero-code/java/agent/performance/).

**Reply by huangtao730** [reply]
yes，This is what I want。Is there any other Documentation?Thanks！

**Reply by steverao** [reply]
> yes，This is what I want。Is there any other Documentation?Thanks！

As far as I know, it's all. In fact, the problem is a bit complicated, the performance depends on many factors. Different instrumentations might have different influences.
